roles/neutron-data-network/tasks/main.yml

```diff
@@ -59,6 +59,45 @@
     neutron-metadata-agent:
       config_files: /etc/neutron/metadata_agent.ini
 
+- name: neutron lbaas config
+  template:
+    src=etc/neutron/neutron_lbaas.conf
+    dest=/etc/neutron/neutron_lbaas.conf
+    mode=0644
```

Comment:
Spacing here is weird. Looks like you're trying to do the YAML hash way
```
template:
  src: etc/neutron/neutron_lbaas.conf
  dest: /etc/neutron/neutron_lbaas.conf
  mode: 0644
```
But you're still using key=value. If you're going to do key=value it'd be more like
```
template:  src=etc/neutron/neutron_lbaas.conf
           dest=/etc/neutron/neutron_lbaas.conf mode=0644
```